.evergreen/config/test/_template-require-api-version.yml

```diff
@@ -0,0 +1,13 @@
+tasks:
+  - name: "test-requireApiVersion-%mongodbVersion%"
+    tags: [ "standalone", "local", "%mongodbVersion%", "versioned_api" ]
```

Comment:
Do I assume correctly that we never use PHP versions for tags? So any "X.Y" strings (and "latest" or "rapid") would only correspond to server versions?

I suppose there's no reason to use a prefix here, as tags are always treated as strings and we're careful to quote these in YAML.

Either way, this is something we could document in a glossary of tags within `architecture.md`.

Reply:
Added a list of tags to architecture.md, which should clear this up.